package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "ng": "ng",
     "compile": "gulp compile",
-    "build": "npm run build --ws",
+    "copy-icon-font": "cp -R ./node_modules/@cdssnc/gcds-fonts/fonts/icons/* ./packages/web/src/assets/fonts/icons",
```

Comment:
I don't think the `cp` command would work when using this repo on a windows machine.

Reply:
Confirmed on my windows desktop that this will prevent the packages from building.

Reply:
Good catch, and thank you for checking on your Windows as well!